Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Checkout.h

```diff
@@ -92,81 +94,56 @@ typedef void (^BUYDataGiftCardBlock)(BUYGiftCard * _Nullable giftCard, NSError *
 - (BUYRequestOperation *)createCheckoutWithCartToken:(NSString *)cartToken completion:(BUYDataCheckoutBlock)block;
 
 /**
- *  Applies a gift card code to the checkout.
- *
- *  @param giftCardCode The gift card code to apply on an existing checkout on Shopify. Note: This is not the same as the gift card identifier.
- *  @param checkout     An existing BUYCheckout on Shopify
- *  @param block        (^BUYDataCheckoutBlock)(BUYCheckout *checkout, NSError *error);
- *
- *  @return The associated BUYRequestOperation
- */
-- (BUYRequestOperation *)applyGiftCardWithCode:(NSString *)giftCardCode toCheckout:(BUYCheckout *)checkout completion:(BUYDataCheckoutBlock)block;
-
-/**
- *  Removes a gift card from the checkout.
- *
- *  @param giftCardCode The BUYGiftCard identifier to remove on an existing checkout on Shopify.
- *  @param checkout     An existing BUYCheckout on Shopify
- *  @param block        (^BUYDataCheckoutBlock)(BUYCheckout *checkout, NSError *error);
- *
- *  @return The associated BUYRequestOperation
- */
-- (BUYRequestOperation *)removeGiftCard:(BUYGiftCard *)giftCard fromCheckout:(BUYCheckout *)checkout completion:(BUYDataCheckoutBlock)block;
-
-/**
- *  Retrieves an updated version of a BUYCheckout from Shopify.
+ *  Updates a given BUYCheckout on Shopify.
  *
  *  Note: There's no guarantee that the BUYCheckout returned will be the same as the one that is passed in.
  *  We recommended using the BUYCheckout returned in the block.
  *
- *  @param checkout The BUYCheckout to retrieve (updated) from Shopify
+ *  Note: A BUYCheckout object with an `orderId` is a completed checkout.
+ *
+ *  @param checkout The BUYCheckout to updated on Shopify
  *  @param block    (^BUYDataCheckoutBlock)(BUYCheckout *checkout, NSError *error);
  *
  *  @return The associated BUYRequestOperation
  */
-- (BUYRequestOperation *)getCheckout:(BUYCheckout *)checkout completion:(BUYDataCheckoutBlock)block;
+- (BUYRequestOperation *)updateCheckout:(BUYCheckout *)checkout completion:(BUYDataCheckoutBlock)block;
 
 /**
- *  Updates a given BUYCheckout on Shopify.
- *
- *  Note: There's no guarantee that the BUYCheckout returned will be the same as the one that is passed in.
- *  We recommended using the BUYCheckout returned in the block.
+ *  Retrieves an updated BUYCheckout.
  *
- *  Note: A BUYCheckout object with an `orderId` is a completed checkout.
- *
- *  @param checkout The BUYCheckout to updated on Shopify
- *  @param block    (^BUYDataCheckoutBlock)(BUYCheckout *checkout, NSError *error);
+ *  @param checkoutToken The checkout token for which to retrieve the updated checkout
+ *  @param block         (^BUYDataCheckoutBlock)(BUYCheckout *checkout, NSError *error);
  *
  *  @return The associated BUYRequestOperation
  */
-- (BUYRequestOperation *)updateCheckout:(BUYCheckout *)checkout completion:(BUYDataCheckoutBlock)block;
+- (BUYRequestOperation *)getCheckoutWithToken:(NSString *)checkoutToken completion:(BUYDataCheckoutBlock)block;
```

Comment:
Nitpick: it's a bit subtle, but we should prefer "For" instead of "With" (the preposition) in most of these method signatures. "With" is best reserved for initialization arguments.